### DISCUSSION TITLE
絕區零抽卡紀錄

### DISCUSSION THREAD

**Discussion by kindson07** [lgou2w/HoYo.Gacha#32] [Q&A] [answered]
怎樣拿紀錄url？？？ 不太懂

**Comment by lgou2w** [comment, marked as answer]
启动工具后，点击左侧导航栏 `调频` 图标进入绝区零页面。再点击页面右上角的 `绳匠` 头像根据指引添加账号。
完成后在页面顶部工具条点击 `读取链接` 按钮。如出现错误等情况请尝试在游戏内打开 `调频记录` 页面后，再重试。

> BTW：目前工具 v0 版本对其他语言没有做兼容，可能会出现图片无法显示错误等等。

**Reply by kindson07** [reply]
内部错误：系統找不到指定的路徑。 (os error 3)
遊戲也開了 近期也沒有清理temp 內的文件

**Reply by kindson07** [reply]
遊戲數據文件夾？是哪個？
![螢幕擷取畫面 2024-07-12 202437](https://github.com/user-attachments/assets/ca5fe295-0961-4929-85c8-8d3b3b398a99)
不是遊戲的安裝位置嗎？

**Reply by lgou2w** [reply]
游戏目录里面的 `ZenlessZoneZero_Data` 文件夹：

如：`X:\ZenlessZoneZero Game\ZenlessZoneZero_Data`
